client/my-sites/plans/current-plan/ecommerce-trial/index.tsx

```diff
@@ -0,0 +1,266 @@
+import { Button, Card } from '@automattic/components';
+import { useLocale } from '@automattic/i18n-utils';
+import { useMediaQuery } from '@wordpress/compose';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import BodySectionCssClass from 'calypso/layout/body-section-css-class';
+import {
+	getCurrentPlan,
+	getECommerceTrialDaysLeft,
+	getECommerceTrialExpiration,
+	isECommerceTrialExpired,
+} from 'calypso/state/sites/plans/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import DoughnutChart from '../../doughnut-chart';
+import FeatureIncludedCard from '../feature-included-card';
+import FeatureNotIncludedCard from '../feature-not-included-card';
+
+import './style.scss';
+
+const ECommerceTrialCurrentPlan = () => {
+	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) ) || -1;
+
+	const { currentPlan, eCommerceTrialDaysLeft, isTrialExpired, eCommerceTrialExpiration } =
+		useSelector( ( state ) => ( {
+			currentPlan: getCurrentPlan( state, selectedSiteId ),
+			isTrialExpired: isECommerceTrialExpired( state, selectedSiteId ),
+			eCommerceTrialDaysLeft: Math.round( getECommerceTrialDaysLeft( state, selectedSiteId ) || 0 ),
+			eCommerceTrialExpiration: getECommerceTrialExpiration( state, selectedSiteId ),
+		} ) );
+
+	const locale = useLocale();
+	const moment = useLocalizedMoment();
+	const translate = useTranslate();
+
+	const [ showAllTrialFeaturesInMobileView, setShowAllTrialFeaturesInMobileView ] =
+		useState( false );
+
+	const viewAllIncludedFeatures = () => {
+		setShowAllTrialFeaturesInMobileView( true );
+	};
+
+	const isMobile = useMediaQuery( '(max-width: 480px)' );
+	const displayAllIncluded = ! isMobile || showAllTrialFeaturesInMobileView;
+
+	const allIncludedFeatures = [
+		{
+			title: translate( 'Priority support' ),
+			text: translate( 'Need help? Reach out to us anytime, anywhere.' ),
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/priority-support.svg',
+			showButton: true,
+			buttonText: translate( 'Ask a question' ),
+		},
+		{
+			title: translate( 'Premium themes' ),
+			text: translate( 'Explore a diverse selection of beautifully designed premium themes.' ),
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/premium-themes.svg',
+			showButton: true,
+			buttonText: translate( 'Browse premium themes' ),
+		},
```

Comment:
Given that I don't think this should be a primary feature, can we remove the `translate()` wrappers?